library/ssl_ciphersuites.c

```diff
@@ -1876,15 +1876,26 @@ int mbedtls_ssl_get_ciphersuite_id( const char *ciphersuite_name )
 
 size_t mbedtls_ssl_ciphersuite_get_cipher_key_bitlen( const mbedtls_ssl_ciphersuite_t *info )
 {
-#if defined(MBEDTLS_CIPHER_C)
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
+    psa_key_type_t key_type;
+    psa_algorithm_t alg;
+    size_t key_bits;
+
+    status = mbedtls_ssl_cipher_to_psa( info->cipher,
```

Comment:
The CI points out that this function is not declared in builds without `MBEDTLS_USE_PSA_CRYPTO`. I suggest fixing the guards around its declaration and definition to `MBEDTLS_PSA_CRYPTO_C` instead - or `MBEDTLS_USE_PSA_CRYPTO || MBEDTLS_SSL_PROTO_TLS1_3`.

Reply:
@mpg only `psa_ssl_status_to_mbedtls` is guarded with `MBEDTLS_USE_PSA_CRYPTO || MBEDTLS_SSL_PROTO_TLS1_3`
`mbedtls_ssl_cipher_to_psa` is only guarded by `MBEDTLS_USE_PSA_CRYPTO`.

So whould I add `|| MBEDTLS_SSL_PROTO_TLS1_3` or switch both to `MBEDTLS_PSA_CRYPTO_C` ?

Reply:
If we already have `MBEDTLS_USE_PSA_CRYPTO || MBEDTLS_SSL_PROTO_TLS1_3` in one place, then let's use that!